manifests/webhook.pp

```diff
@@ -61,11 +61,31 @@
     before => File['webhook_init_script'],
   }
 
+  if $service_file =~ /systemd/ {
+
+    $_service_file_mode = '0644'
+
+    exec { 'webhook_systemd_reload':
```

Comment:
Hi. we try to use https://github.com/camptocamp/puppet-systemd on all Vox Pupuli modules. puppet-systemd provides an exec that can be notified, please use that instead of defining a new one.